src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
 // write code here
+const wall = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
+
+spider.style.left =
+  Math.round(wall.offsetWidth - spider.clientWidth) / 2 + 'px';
+
+spider.style.top =
```

Comment:
This line is incomplete. It seems like you intended to set the 'top' style property of the spider element, but the assignment operator '=' is missing a value. Ensure that the calculation on the next line is part of this assignment.